book/src/clauses/coherence.md

```diff
@@ -166,7 +166,7 @@ The conclusion from all of this is that it is perfectly safe to rule out impls t
 
 **Downstream Crates:** Downstream crates come into play because all traits in upstream crates and in the current crate can potentially be implemented by downstream crates using the forms allowed by the orphan rules. In essence, we always need to assume that downstream crates will implement traits in all ways that compile.
 
-## Discussion: Modelling the Overlap Check
+## Discussion: Modeling the Overlap Check
```

Comment:
Looks like one or 2 L's are both fine, depending on whether it's American English or not. But I think we default to American English in most places, so :+1:

Reply:
Oh I had no idea that it was correct in non American English. I can leave it as it was.